### DISCUSSION TITLE
How can I make a linear progress bar vertically?

### DISCUSSION THREAD

**Discussion by David-He** [quasarframework/quasar#9802] [General - Components / Directives / etc]
How can I make a linear progress bar vertically?
Or just can show it horizontally?

**Comment by jnui** [comment]
you could try using just CSS

transform: rotate(
-90deg
)

**Reply by David-He** [reply]
Thanks @jnui . Yes the bar  standed up. Great. But I am not familiar with CSS. then how to let the vertical progress bar's height align to parent's height?
